actions/build-push-to-dockerhub/README.md

```diff
@@ -31,13 +31,14 @@ jobs:
 
 ## Inputs
 
-| Name | Type | Description |
-|------|------|-------------|
-| `context` | String | Path to the Dockerfile (default: `.`) |
-| `platforms` | List | List of platforms the image should be built for (e.g. `linux/amd64,linux/arm64`) |
-| `push` | Bool | Push the generated image (default: `false`) |
-| `repository`| String | Docker repository name |
-| `tags` | List | Tags that should be used for the image (see the [metadata-action][mda] for details) |
+| Name         | Type   | Description                                                                         |
+|--------------|--------|-------------------------------------------------------------------------------------|
+| `context`    | String | Path to the Dockerfile (default: `.`)                                               |
+| `platforms`  | List   | List of platforms the image should be built for (e.g. `linux/amd64,linux/arm64`)    |
+| `push`       | Bool   | Push the generated image (default: `false`)                                         |
+| `repository` | String | Docker repository name                                                              |
+| `tags`       | List   | Tags that should be used for the image (see the [metadata-action][mda] for details) |
+| `file`       | String | Path and filename of the dockerfile to build from.                                  |
```

Comment:
Given https://github.com/grafana/shared-workflows/pull/47#issuecomment-1960907637, I think it's useful to tell people that this path is resolved relative to the workspace and not the context directory.